frontend/css/main.less

```diff
@@ -43,6 +43,7 @@
 @import "scroll-container.less";
 @import "flairs.less";
 @import "utilities.less";
+@import "link-listens-modal.less";
```

Comment:
With CSS classes suggestions below, you'll be able to remove this line and the new file itself.